client/lib/plans/constants.js

```diff
@@ -424,6 +434,59 @@ const getPlanBusinessDetails = () => ( {
 	getHiddenFeatures: () => [ FEATURE_AUDIO_UPLOADS ],
 } );
 
+const getPlanEcommerceDetails = () => ( {
+	group: GROUP_WPCOM,
+	type: TYPE_ECOMMERCE,
+	getTitle: () => i18n.translate( 'eCommerce' ),
+	getAudience: () => i18n.translate( 'Best for eCommerce sites' ),
+	getBlogAudience: () => i18n.translate( 'Best for online stores' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Best for organizations and online stores.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).